### DISCUSSION TITLE
怎么感觉人好少

### DISCUSSION THREAD

**Discussion by aino2333** [share121/inter-knot#5352] [绝区零]
如题，再试一试发帖

**Comment by Tu-Doou** [comment]
人就是很少

**Comment by loveSM0708** [comment]
门槛还是有一定的，宣发也不够

**Comment by TNCNET** [comment]
一个很明显的是门槛
•知道没法直连GitHub这些网站的时候该怎么办；
•正确的打开GitHub；
•注册账号，了解基本操作；
•绑定设备码，配置软件；
•对于账号安全的部分，还需要2FA（如验证码/totp/otp）和相关设置。（但是是另一回事了）

另一个，宣传也不到位。

**Comment by mephistophelesCLOUD** [comment]
因为人就是少啊
